README.md

```diff
@@ -251,23 +281,27 @@ Orbit 4 is called Observers. These are folks who’ve read our community's blog
 
-## Orbit 2 - Fans
+## Orbit 2 - Contributors
 
-Orbit 2 is called Fans. These folks have graduated from being just Users by taking on more time-intensive, higher impact roles in the community. They might be moderating the forum, guiding the chat room, giving talks, or hosting small events. They're consistent, recognizable faces who embody the culture and actions the community expects.
+People in Orbit 2 are called Contributors. These folks have graduated from being just Participants by taking on more time-intensive, higher impact roles in the community. They might be moderating the forum, guiding the chat room, giving talks, or hosting small events. They're consistent, recognizable faces who embody the culture and actions the community expects.
 
-Motivating and engaging with Fans requires a higher touch than Users or Observers. Fans will appreciate activities that pair them with prominent members of the community and offer deeper learning or development experiences. As a Fan, they’re making a strong contribution to the community, and the recognition and opportunities provided to them should be commensurate with that.
+Motivating and engaging with Contributors requires a higher touch than Participants or Observers. 
 
-## Orbit 1 - Ambassadors
+They will appreciate activities that pair them with prominent members of the community and offer deeper learning or development experiences. As a Contributor, they’re making a strong contribution to the community, and the recognition and opportunities provided to them should be commensurate with that.
 
-Orbit 1 is reserved for the inner circle, the Ambassadors. These individuals have very high love for the community, taking on important challenges and playing a leadership role. They're known to many other members, who look to them for guidance and help. They may head one or more large groups inside of the community.
+## Orbit 1 - Core
 
-The relationship with ambassadors is close, personal, and transparent. Ambassadors appreciate getting early access to products and events, access to people at the Team level, and opportunities that are specific to their individual goals. For example, you could use your network to help an Ambassador find a new job, or if they have a large audience you could offer to sponsor or contribute to their newsletter.
+Orbit 1 is reserved for the inner circle, the Core members. These individuals have very high love for the community, taking on important challenges and playing a leadership role. They're known to many other members, who look to them for guidance and help. They may head one or more large groups inside of the community.
+
+The relationship with the core is close, personal, and transparent. They appreciate getting early access to products and events, access to people at the Team level, and opportunities that are specific to their individual goals. 
```

Comment:
I really like those insights into how to engage with people in the different Orbit levels ❤️